tuf/repository_lib.py

```diff
@@ -56,6 +55,8 @@
 import iso8601
 import six
 
+import securesystemslib.storage
```

Comment:
Could this go up with the other ssl imports?

Reply:
Good catch, thanks. I've fixed this and squashed the change into "Port to new securesystemslib w abstract filesystem" 0214302